client/devdocs/design/test/gutenberg.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Gutenberg from '../gutenberg';
+import Main from 'components/main';
+import ReadmeViewer from 'components/readme-viewer';
+
+describe( 'Gutenberg', () => {
+	test( 'should render', () => {
+		const wrapper = shallow( <Gutenberg/> );
+
+		expect( wrapper ).toMatchSnapshot();
+	} );
+
+	test( 'should have a main component', () => {
+		const wrapper = shallow( <Gutenberg/> );
+		const main = wrapper.find( Main );
+
+		expect( main ).toHaveLength( 1 );
+		expect( main.hasClass( 'devdocs devdocs__gutenberg' ) ).toBe( true );
+	} );
+
+	test( 'should have a document head component', () => {
+		const wrapper = shallow( <Gutenberg/> );
+		const documentHead = wrapper.find( DocumentHead );
+
+		expect( documentHead ).toHaveLength( 1 );
+		expect( documentHead.props().title ).toBe( 'Gutenberg Blocks' );
+	} );
+
+	test( 'should have a readme viewer component', () => {
+		const wrapper = shallow( <Gutenberg/> );
+		const readmeViewer = wrapper.find( ReadmeViewer );
+
+		expect( readmeViewer ).toHaveLength( 1 );
+		expect( readmeViewer.props().readmeFilePath ).toBe( '/client/devdocs/gutenberg/README.md' );
+	} );
```

Comment:
I'm not convinced on the value of these tests. I suspect that we'd know if the sections are missing because someone would intentionally be deleting them. further, what _behaviors_ are these tests verifying? if it were me I'd probably leave them out.

Reply:
you're right, this is not checking any behavior. 

I just have the habit of writing unit tests for everything but, as pointed [here](https://wpcalypso.wordpress.com/devdocs/docs/testing/unit-tests.md), that's not the goal if we want to strike the right balance between covering expected and unexpected behaviors, speedy execution and code maintenance.